### ISSUE TITLE
Incorrect main/module/exports in `package.json` in v2.1.1

### ISSUE THREAD

**Issue by Tanimodori** [ngVue/ngVue#144]
The fix of #143 brings a new bug, when I trying to build the project, the following message emerges:

```
[commonjs--resolver] Failed to resolve entry for package "ngVue". The package may have incorrect main/module/exports specified in its package.json: No known conditions for "." entry in "ngVue" package
error during build:
Error: Failed to resolve entry for package "ngVue". The package may have incorrect main/module/exports specified in its package.json: No known conditions for "." entry in "ngVue" package
```

The current `package.json` is incorrect as follows:

```jsonc
{
  "name": "ngVue",
  "author": "Doray Hong <hongduhui@gmail.com>",
  "version": "2.1.1",
  "description": "Use Vue Components in Angular 1.x",
  "main": "build/index.js",
  "module": "build/index.esm.js",
  "exports": {
    "*": {  // <------------------------ SHOULD BE "."
      "require": "./build/index.js",
      "import": "./build/index.esm.js"
    }
  },
  // ...
}
```

Change "*" to "." will fix this problem.

Current workaround is the same approach as in #143

**Comment by jaredmcateer** [comment]
Sorry about that. this should be fixed now.

**Comment by Tanimodori** [comment]
This issue is fixed in v2.1.3, thanks!
